source/stylesheets/_constants.scss

```diff
@@ -25,6 +25,10 @@ $def-ultralightgrey: #f4f4f4;
 $def-yellow: #ffc500;
 $def-yellow-light: #ffc50066;
 
+$def-grape: #492f5c;
+$def-newgrey: #dddddd;
```

Comment:
Use `$def-lightgrey` instead of `$def-newgrey` for now.
Use `$def-darkgrey` instead of `$def-new-darkgey` for now.